src/hashtable.py

```diff
@@ -51,7 +51,13 @@ def insert(self, key, value):
 
         Fill this in.
         '''
-        pass
+        hashed = self._hash_mod(key)
+        if self.storage[hashed] == None:
+            self.storage[hashed] = LinkedPair(key, value)
+        else:
+            collided = LinkedPair(key, value)
+            collided.next = self.storage[hashed]
+            self.storage[hashed] = collided
```

Comment:
Added comments to the code so that you can have a better understanding of how to store the value with the given key using insert function.
```
hashed = self._hash_mod(key)

        if self.storage[hashed] is not None:
            curr = self.storage[hashed]
            while True:
                #Overwrite values when the key is same
                if curr.key is key:
                    curr.value = value
                    break
                #Check if there is another pair next, if not add this one to the chain
                if curr.next is None:
                    curr.next = LinkedPair(key, value)
                    self.count += 1
                    break
                #Move to the next pair
                elif curr.next is not None:
                    curr = curr.next
                #Print error warning
                else:
                    print('Warning! Something is already written on this index.')
        else:
            self.storage[hashed] = LinkedPair(key, value)
            self.count += 1
```